docs/how_to/compose_quantum_circuits.rst

```diff
@@ -0,0 +1,185 @@
+
+:meth:`~.QuantumCircuit.compose`
+------------------------------------------------
+
+In order to combine two circuits with :meth:`~.QuantumCircuit.compose`, you only have to specify the circuit you want to insert. That way the qubits and bits of the smaller circuit will be included into the first qubits and bits of the bigger one in the original order they had. 
+
+By default, :meth:`~.QuantumCircuit.compose` does not change the circuit to which it is applied but returns the composed circuit. This can be changed by setting the ``inplace`` argument to ``True``.
+
+.. testcode::
+
+    qc3 = qc2.compose(qc1)
+    print(qc3.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+         ┌───┐     ┌───┐        
+    q_0: ┤ Y ├──■──┤ H ├──■─────
+         ├───┤  │  └───┘┌─┴─┐┌─┐
+    q_1: ┤ X ├──┼───────┤ X ├┤M├
+         └───┘  │       └───┘└╥┘
+    q_2: ───────┼─────────────╫─
+              ┌─┴─┐ ┌─┐       ║ 
+    q_3: ─────┤ X ├─┤M├───────╫─
+              └───┘ └╥┘       ║ 
+    c: 2/════════════╩════════╩═
+                     1        0 
+
+If you want to insert the qubits and bits into specific positions in the bigger circuit, you can use the ``qubits`` and ``bits`` arguments.
+
+.. testcode::
+
+    qc4 = qc2.compose(qc1, qubits=[3,1], clbits=[1])
+    print(qc4.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+         ┌───┐                     
+    q_0: ┤ Y ├──■──────────────────
+         ├───┤  │          ┌───┐┌─┐
+    q_1: ┤ X ├──┼──────────┤ X ├┤M├
+         └───┘  │          └─┬─┘└╥┘
+    q_2: ───────┼────────────┼───╫─
+              ┌─┴─┐┌─┐┌───┐  │   ║ 
+    q_3: ─────┤ X ├┤M├┤ H ├──■───╫─
+              └───┘└╥┘└───┘      ║ 
+    c: 2/═══════════╩════════════╩═
+                    1            1 
+
+You can also apply the gates from the smaller circuit before those of the bigger one setting the ``front`` argument to ``True``.
+
+.. testcode::
+
+    qc5 = qc2.compose(qc1, front=True)
+    print(qc5.draw())
+
+.. testoutput::
+    :options: +NORMALIZE_WHITESPACE
+
+         ┌───┐     ┌───┐             
+    q_0: ┤ H ├──■──┤ Y ├───────■─────
+         └───┘┌─┴─┐└┬─┬┘┌───┐  │     
+    q_1: ─────┤ X ├─┤M├─┤ X ├──┼─────
+              └───┘ └╥┘ └───┘  │     
+    q_2: ────────────╫─────────┼─────
+                     ║       ┌─┴─┐┌─┐
+    q_3: ────────────╫───────┤ X ├┤M├
+                     ║       └───┘└╥┘
+    c: 2/════════════╩═════════════╩═
+                     0             1 
+
+:meth:`~.QuantumCircuit.append`
+-----------------------------------------------
+
+In order to combine two circuits with :meth:`~.QuantumCircuit.append`, you have to specify the circuit you want to add and the qubits and classical bits (if there are any) into which you want the circuit to be inserted.
+
+This method changes the circuit to which it is applied instead of returning another one.
+
+.. testcode::
+
+    qc2.append(qc1, qargs=[3,1], cargs=[1])
+    qc2.draw(cregbundle=False)
```

Comment:
What does `cregbundle=False` do? To show individual classical bits instead of showing them in one line? Can you add a comment saying that? Otherwise people will wonder what it does.

Reply:
Yes, that's exactly what it does. However I might need to check whether this is still necessary. It's possible that it's no longer needed.